### PR TITLE
BartWorks

### PR DESCRIPTION
**Pull request by bartimaeusnek**
My stuff (GT2 LESU, GT2 Destrucorpack, GT2 Rockcutter, GT2 Teslastaff)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -58,6 +58,8 @@ minecraft.version = "1.7.10-10.13.4.1614-1.7.10"
 minecraft {
     version = "1.7.10-10.13.4.1614-1.7.10"
     runDir = "$projectDir"
+	replaceIn "MainMod.java"
```
Comment (draknyte1) [inline build.gradle]: I have 0 idea what this does.

Comment (bartimaeusnek) [reply build.gradle]: replaces the version definition in my mod/submodule

Comment (bartimaeusnek) [reply build.gradle]: https://github.com/bartimaeusnek/bartworks-code/blob/9cef3f2d33dc444afae119badfbdccc55415b30d/github/bartimaeusnek/bartworks/MainMod.java#L25

Comment (bartimaeusnek) [reply build.gradle]: Yupp, basically the same method you used for GT++ Australia, makes it easier to to keep the code seperated and your code untouched.

**Comment by e99999** [comment]
@draknyte1 you gonna do this? we are planning on trying to add GT4 features into GT5uBDE. but i dont want overlap with ++ if this gets ported in?

**Pull request closed without merging**